examples/advanced/gnn/requirements.txt

```diff
@@ -1,3 +1,5 @@
+nvflare
+torch
+tensorboard
 scikit_learn
```

Comment:
Thanks.  I agree that we might need to quote the steps in the official installation document if it's available.

Reply:
sounds good! moved main torch_geometric here